ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -64,6 +64,46 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-conc-builds
   timeout: 5h0m0s
+- as: sdn-to-ovn-24nodes
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
+      OVN_SDN_MIGRATION_TIMEOUT: 3600s
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - ref: ovn-sdn-migration
+- as: sdn-to-ovn-120nodes
+  cron: 0 14 * * 0
+  steps:
+    cluster_profile: aws-perfscale-qe
```

Comment:
Anything over 24 nodes needs to use the aws-perfscale profile

Reply:
Nice catch - will fix it up

Reply:
Updated